WebContent/js/containers/ContentViewer.js

```diff
@@ -152,7 +194,9 @@ export class ContentViewer extends React.Component {
     renderSubheader() {
         return (
             <div style={{ height: '38px' }}>
-                { this.renderTabs() }
+                <div role="tablist" aria-label="Open Jobs">
```

Comment:
I think this label should be "Open Job output files"

Reply:
changed label